plugins/inputs/lvm/sample.conf

```diff
@@ -2,3 +2,12 @@
 [[inputs.lvm]]
   ## Use sudo to run LVM commands
   use_sudo = false
+
+  ## The default location of the pvs binary can be overridden with:
+  pvs_binary = "/usr/sbin/pvs"
+
+  ## The default location of the vgs binary can be overridden with:
+  vgs_binary = "/usr/sbin/vgs"
+
+  ## The default location of the lvs binary can be overridden with:
+  lvs_binary = "/usr/sbin/lvs"
```

Comment:
Can you please comment those options as those should be the defaults!?

Reply:
Done in commit https://github.com/influxdata/telegraf/pull/12725/commits/a24a4faaf86e7f5060d50d4ff2e79f74fecee360